projects/core/src/lib/users/user.service.ts

```diff
@@ -325,6 +330,20 @@ export class UserService {
                 this.sessionQuota$.next(quota);
             });
     }
+
+    private checkOidcParametersAndLogin(): boolean {
+        const params = new URLSearchParams(window.location.search);
+        const sessionState = params.get('session_state');
+        const code = params.get('code');
+        const state = params.get('state');
```

Comment:
cf. chat, removal of parameters

Reply:
add09e802673f1b7d52dc319be3caa6f2b1e451e